students/hwijeong/westagram/users/validators.py

```diff
@@ -0,0 +1,13 @@
+import re
+
+from django.core.exceptions import ValidationError
+
+def isEmailValid(email):
+    REGEX_EMAIL = '([A-Za-z0-9]+[.-_])*[A-Za-z0-9]+@[A-Za-z0-9-]+(\.[A-Z|a-z]{2,})+'
+    if not re.fullmatch(REGEX_EMAIL, email) or email == '':
```

Comment:
`email == ''` 이 로직은 정규표현식에서 걸러내야 하는게 맞지 않을까요? 아래도 마찬가지입니다.
```suggestion
    if not re.fullmatch(REGEX_EMAIL, email):
```